composer.json

```diff
@@ -14,8 +14,8 @@
         }
     ],
     "require": {
-        "php": "^7.0",
-        "phpspec/phpspec": "6.*"
+        "php": "^7.0 | ^8.0",
```

Comment:
@JellyBellyDev 
Would you mind to set the lower php version to `7.3` and update `travis.yml` file accordingly?

Moreover we should test all the things with `--PREFER-LOWEST` also; would you mind to add it to building matrix too?

Reply:
@DonCallisto done, but i don't see travis pipeline... could you tell me why?

Reply:
Umh, my bad, maybe we didn't setted it up ever. 
I will create the travis ci, in order to check if everything's ok, in the upcoming days.
Let's keep in touch. For the moment, thanks for this PR.

Reply:
I've enabled travis for this. Could you please try to trigger the build (maybe by slightly change the commit message?)

Reply:
done! ;)